spec/factories/move.rb

```diff
@@ -10,5 +10,7 @@
     status { 'requested' }
     additional_information { 'some more info about the move that the supplier might need to know' }
     move_type { 'court_appearance' }
+    cancellation_reason { 'other' }
+    cancellation_reason_comment { 'other reason' }
```

Comment:
I don't think this is changing the tests. The new attributes should be declared in the `MovesSerializer` and in param whitelists - https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/controllers/api/v1/moves_controller.rb#L47-L53